app/src/main/java/org/connectbot/util/VolumePreference.java

```diff
@@ -56,7 +57,13 @@ protected View onCreateDialogView() {
 		sb.setMax(100);
 		sb.setProgress((int) (getPersistedFloat(
 				PreferenceConstants.DEFAULT_BELL_VOLUME) * 100));
-		sb.setPadding(10, 10, 10, 10);
+
+		if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+			sb.setPadding(75, 70, 75, 10);
```

Comment:
Could this be moved to a resource? I didn't realize this was in the java code. It'd be best to put it in an XML layout, but it looks like this doesn't lend itself to that. Instead you could just add a `<dimen>` resource.

In a resource file such as `res/values/dimens.xml` you could add:
```
<dimen name="volume_dialog_padding_horizontal">75dp</dimen>
<dimen name="volume_dialog_padding_vertical">10dp</dimen>
```

Then you could do:
```
int paddingHorizontal = (int) getResources().getDimension(R.dimen.volume_dialog_padding_horizontal);
int paddingVertical = (int) getResources().getDimension(R.dimen.volume_dialog_padding_vertical);
sb.setPadding(paddingHorizontal, paddingVertical, paddingHorizontal, paddingVertical);
```